docs/snippets/react/my-component-story-configure-viewports.js.mdx

```diff
@@ -5,23 +5,26 @@ import React from 'react';
 
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 
+import MyComponent from './MyComponent';
```

Comment:
```suggestion
import { MyComponent } from './MyComponent';
```

Reply:
In this case, if you don't mind, I'll leave it as is. I wrote the component with a default export, but I can't guarantee if the reader uses named exports per his/her development patterns.